### DISCUSSION TITLE
使用bisenet v2 训练自己的数据集 miou太差什么原因？

### DISCUSSION THREAD

**Discussion by AlbertYZM** [MaybeShewill-CV/bisenetv2-tensorflow#51] [Q&A]
使用bisenet v2 训练cityscapes数据集miou达到了0.94 但是训练自己的数据集miou 一直在0.004左右
![2021-05-28 10-38-25屏幕截图](https://user-images.githubusercontent.com/69412525/119921592-ddb17800-bfa0-11eb-8707-b2ffb8d07baf.png)

**Comment by MaybeShewill-CV** [comment]
@AlbertYZM 1.首先还是检查下标签有没有什么问题 2.其次就是调参数吧. 不过按照经验,　参数再差miou也不会这么低, 还是先检查下数据，配置文件里的类别数之类的有没有写错:)
